examples/pyplots/pyplot_two_subplots.py

```diff
@@ -1,10 +1,11 @@
 """
-===================
-Pyplot Two Subplots
-===================
+========================
+Two subplots with pyplot
```

Comment:
Should we use "using pyplot" instead of "with pyplot"?
```suggestion
Two subplots using pyplot
```
Also for the other titles?

Reply:
sure